image_provisioner/playbooks/roles/docker-config/tasks/main.yaml

```diff
@@ -38,6 +38,14 @@
    - name: reset docker storage
      command: atomic storage reset
   when: atomic | default(False) | bool
+
+- block:
+   - name: format the partition
+     command: mkfs.xfs -f {{ docker_storage_device }}
+
+   - name: update fstab
+     shell: echo "{{ docker_storage_device }} /var/lib/docker/overlay2                    xfs   defaults        0 0" >> /etc/fstab
```

Comment:
Need to make sure selinux labels are intact at the end of this process.  /var/lib/docker/overlay2 should be labelled container_share_t.   An early AMI with this change required restorecon to re-label the dir.

Reply:
Good find @mffiedler, maybe have an extra reboot and auto-relabel the whole thing as an insurance policy?

Reply:
@chaitanyaenr  @wabouhamad 1)  touch /.autorelabel  2) reboot